level05/README.md

```diff
@@ -1,40 +1,29 @@
-# Level 05 - Divergence and Phases
+# Level 05 - Phases and Transitions
 
-You may have noticed that in the process of building the model and various
-commands, we're building another state machine! One that could almost be a
-replica of the thing we're trying to test. This is a problem: our goal is to
-prove that the thing we're testing operates correctly, not duplicate it.
+You may have noticed that in the process of building the model and
+various commands, we're building another state machine! This is a
+problem: our goal is to prove that the thing we're testing operates
+correctly, not duplicate it. How do we create _just enough_ of a model
+so we're able to write useful tests, without creating a crummy
+duplicate of the thing we're testing? This is where state machine
+testing becomes an art.
 
-So, how do we create _just enough_ of a model so we're able to write useful
-tests, without creating a crummy duplicate of the thing we're testing? This is
-where state machine testing becomes an art.
+The goal: construct a minimal the model that can still support
```

Comment:
"minimal the model" bonus word! +10 points

Reply:
nnnnnnnnnnnnnnnnnnnnnnnnnnnnn